### ISSUE TITLE
Support documents fewer than expected

### ISSUE THREAD

**Issue by bugtig** [facebookresearch/ELI5#14]
Hello, 
I really appreciate your efforts in building this fantastic dataset and the code release.

After following all the steps of the data creation, I got train/val/test files of 1.3gb, 56mb, 140mb, with split sizes 234212/9929/24819. Do these values look okay?

Additionally, while the generated contexts appear to be good, I found that during the support docs download stage, each QA pair does not end up with 100 supporting documents from common crawl. Most of my data has on average ~50 supporting documents rather than the 100. 

Is this correct or was there something wrong with my run?

**Comment by mnskim** [comment]
Thank you!

**Comment by bugtig** [comment]
Thank you!!
